lib/hop_swipe_cards.dart

```diff
@@ -66,38 +66,38 @@ class HopSwipeCards extends StatefulWidget {
   final List<Alignment> _cardAligns = [];
 
   //If you have points left, you can swipe the card
-  final bool _canSwipe;
+  final bool? _canSwipe;
 
-  final VoidCallback cantSwipeLikeWhenNoPointsCallback;
+  final VoidCallback? cantSwipeLikeWhenNoPointsCallback;
 
-  final OnRestrictLeftSwipeCallBack _onRestrictLeftSwipeCallBack;
+  final OnRestrictLeftSwipeCallBack? _onRestrictLeftSwipeCallBack;
 
-  final VoidCallback _triedSwipeLeftDuringRestriction;
+  final VoidCallback? _triedSwipeLeftDuringRestriction;
 
-  final CurrentIndexInDisplay _currentIndexInDisplay;
+  final CurrentIndexInDisplay? _currentIndexInDisplay;
 
   @override
   _HopSwipeCardsState createState() => _HopSwipeCardsState();
 
   HopSwipeCards(
-      {@required SwipeCardBuilder cardBuilder,
-      @required int totalNum,
+      {required SwipeCardBuilder cardBuilder,
+      required int totalNum,
       int currentStack = 5,
       Duration animationDuration = const Duration(milliseconds: 260),
       double swipeEdge = 2.6,
-      double maxWidth,
-      double maxHeight,
-      double minWidth,
-      double minHeight,
-      Widget noMoreSwipeCardsLeft,
-      bool isPointsLeft,
+      required double maxWidth,
+      required double maxHeight,
+      required double minWidth,
+      required double minHeight,
+      required Widget noMoreSwipeCardsLeft,
+      bool? isPointsLeft,
```

Comment:
- Why can `isPointsLeft` be null? it is bool type.

Reply:
@masashi-sutou 
I fixed it 🙏